.gitattributes

```diff
@@ -83,6 +83,6 @@ cmd/workspace/warehouses/warehouses.go linguist-generated=true
 cmd/workspace/workspace-bindings/workspace-bindings.go linguist-generated=true
 cmd/workspace/workspace-conf/workspace-conf.go linguist-generated=true
 cmd/workspace/workspace/workspace.go linguist-generated=true
-
-# Hide diff for Go structs generated from databricks terraform provider schema
 bundle/internal/tf/schema/*.go linguist-generated=true
+go.sum linguist-generated=true
+bundle/schema/docs/bundle_descriptions.json linguist-generated=true
```

Comment:
We only ran code gen once since this change was made. I suspect it survived because @andrewnester noticed it. 

We need to append it, otherwise on autogen the manually added bits are truncated,